soroban-sdk/src/storage.rs

```diff
@@ -139,6 +139,14 @@ impl Storage {
         }
     }
 
+    /// Returns the maximum ledger sequence that an entry can live to
+    /// (inclusive).
+    pub fn max_expiration(&self) -> u32 {
+        internal::Env::get_max_expiration_ledger(&self.env)
```

Comment:
I've just realized this doesn't work well with the bump interface, which operates on 'relative' ledger counts, not the absolute ledger seq. I think we should at least compensate for that in the SDK (subtract current ledger seq), but also maybe change the host fn semantics to not do unnecessary extra work.

Reply:
Ah, it returns the max actual seq, not the max count. Thanks for catching that.

Reply:
Fixed in b45806e.